### PR TITLE
Bug Fix: Updating the fitter to work with ROOT 6 and fix bug 62

### PR DESCRIPTION
**Pull request by BenjaminRS**
## Description

This is a simple bug fix to the fitting mechanism of the scurve analyser. It implements the bug fix proposal from Brian.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context

Fixes https://github.com/cms-gem-daq-project/gem-plotting-tools/issues/62

## How Has This Been Tested?

Ran an analysis of previously taken scurve data before and after the fix. Post fix the fit does not crash and creates the appropriate fits.

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by bdorney** [comment]
@jsturdy your `rpm` prep work is based off `master` correct?  If so then I will approve and merge this into `develop` and it will be incorporated into a future release.

**Comment by jsturdy** [comment]
Does it pass `ROOT` 5 and 6 tests?
The Docker container will contain the following `ROOT` versions for setting up the testing:
* `v5.34.28` (`slc6`/`cc7`)
* `v5.34.36` (`slc6`/`cc7`)
* `v6.10.08 ` (`cc7` only)
* `v6.12.06` (`cc7` only)

File: fitting/fitScanData.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def fit(self):
             for ch in range(0,128):
                 if self.isDead[vfat][ch]:
                     continue # Don't try to fit dead channels
+		elif not (self.scanHistos[vfat][ch].Integral() > 0):
```
Comment (jsturdy) [inline fitting/fitScanData.py]: Seems to be some whitespace/alignment difference, so I suspect a `TAB` is hanging around
Can you convert all `TAB`s to the appropriate number of `space`s?

Comment (BenjaminRS) [reply fitting/fitScanData.py]: sigh: done

**Comment by BenjaminRS** [comment]
Tested in:
ROOT 6.12/04 via gem904qc8daq
ROOT 5.34/36 via gem904daq03 (with Python 2.6.6 and numpy 1.11.3)
Tests appear to be succesful

**Pull request merged**